frontend/src/components/SimpleRange/SimpleRange.js

```diff
@@ -2,38 +2,46 @@ import { styled } from '@mui/material/styles'
 import LinearProgress, { linearProgressClasses } from '@mui/material/LinearProgress'
 import PropTypes from 'prop-types'
 
-const GradientProgress = styled(LinearProgress)(({ theme, value }) => ({
-  [`& .${linearProgressClasses.bar}`]: {
-    overflow: 'hidden',
-    backgroundColor: 'transparent',
-    borderRadius: 1,
-    '&:before': {
-      content: '""',
-      display: 'block',
-      width: '100%',
-      height: '100%',
-      borderRadius: theme.shape.borderRadius,
-      transition: 'transform .4s linear',
-      transform: `translateX(${100 - value}%)`,
-      background: `linear-gradient(270deg, ${theme.palette.canary.main} 0%, ${theme.palette.chestnutRose.main} 50%, ${theme.palette.ultramarine.main} 100%)`, // eslint-disable-line
+const nonDomPropsList = ['colorScheme']
+const GradientProgress = styled(LinearProgress, { shouldForwardProp: name => !nonDomPropsList.includes(name) })(
+  ({ theme, value, colorScheme }) => ({
+    [`& .${linearProgressClasses.bar}`]: {
+      overflow: 'hidden',
+      backgroundColor: 'transparent',
+      borderRadius: 1,
+      '&:before': {
+        content: '""',
+        display: 'block',
+        width: '100%',
+        height: '100%',
+        borderRadius: theme.shape.borderRadius,
+        transition: 'transform .4s linear',
+        transform: `translateX(${100 - value}%)`,
+        background: `linear-gradient(270deg, ${colorScheme?.[0] || theme.palette.canary.main} 0%, ${
+          colorScheme?.[1] || theme.palette.chestnutRose.main
+        } 50%, ${colorScheme?.[2] || theme.palette.ultramarine.main} 100%)`, // eslint-disable-line
+      },
     },
-  },
-}))
+  }),
+)
 
-const SimpleRange = ({ min, max, value, style, color }) => {
+const SimpleRange = ({ min, max, value, style, color, colorScheme, height, sx = {}, ...otherProps }) => {
   const Range = style === 'gradient' ? GradientProgress : LinearProgress
   const percentage = Math.min(1, (value - min) / (max - min || 1))
   return (
     <Range
       value={percentage * 100}
       sx={{
-        height: 24,
+        height: height || 24,
```

Comment:
Instead of supporting arbitrary heights, I'd prefer that SimpleRange take a `size` prop with `size='medium'` giving a height of 24 and `size='small'` giving a height of 14.